app/assets/javascripts/admin/views/project/project_list_view.js

```diff
@@ -1,67 +1,196 @@
+                project.priority.toString(),
+              )}
+              render={(priority, project: APIProjectType) =>
+                `${priority} ${project.paused ? "(paused)" : ""}`}
+            />
+            <Column
+              title="Location"
+              dataIndex="assignmentConfiguration"
+              key="assignmentConfiguration"
+              sorter={Utils.localeCompareBy(
+                (project: APIProjectType) => project.assignmentConfiguration.location,
+              )}
+              render={assignmentConfiguration =>
+                <Tag color={TemplateHelpers.stringToColor(assignmentConfiguration.location)}>
+                  {assignmentConfiguration.location}
+                </Tag>}
+            />
+            <Column
+              title="Owner"
+              dataIndex="owner"
+              key="owner"
+              sorter={Utils.localeCompareBy((project: APIProjectType) => project.owner.lastName)}
+              render={owner =>
+                owner.email ? `${owner.firstName} ${owner.lastName} (${owner.email})` : "-"}
+            />
+            <Column
+              title="Open Assignments"
+              dataIndex="numberOfOpenAssignments"
+              key="numberOfOpenAssignments"
+              sorter={Utils.localeCompareBy((project: APIProjectType) =>
+                project.numberOfOpenAssignments.toString(),
+              )}
+            />
+            <Column
+              title="Expected Time"
+              dataIndex="expectedTime"
+              key="expectedTime"
+              sorter={Utils.localeCompareBy((project: APIProjectType) =>
+                project.expectedTime.toString(),
+              )}
+              render={expectedTime => `${parseInt(expectedTime / 60000)}m`}
+            />
+            <Column
+              title="Action"
+              key="actions"
+              render={(__, project: APIProjectType) =>
+                <span>
+                  <a
+                    href={`/annotations/CompoundProject/${project.name}`}
+                    title="View all Finished Tracings"
+                  >
+                    <Icon type="eye-o" />View
+                  </a>
+                  <br />
+                  <a href={`/projects/${project.name}/edit`} title="Edit Project">
+                    <Icon type="edit" />Edit
+                  </a>
+                  <br />
+                  <a href={`/projects/${project.name}/tasks`} title="View Tasks">
+                    <Icon type="book" />Tasks
+                  </a>
+                  <br />
+                  <a
+                    href={`/api/projects/${project.name}/download`}
+                    title="Download all Finished Tracings"
+                  >
+                    <Icon type="download" />Download
+                  </a>
+                  <br />
+                  {project.owner.email === app.currentUser.email
+                    ? <a href="#">
```

Comment:
Project deletion doesn't work yet, you'll probably need to add a click handler here that sends the delete request or there is a specific delete route, I don't know.

Reply:
Ups. I missed that after cherry picking some commits. Should be restored now.